shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -116,13 +116,24 @@
               .desc("reverse order while sorting")
               .build();
 
+  private static final Option TIMESTAMP_OPTION =
+      Option.builder()
+          .required(false)
+          .longOpt("timestamp")
+          .hasArg(true)
+          .desc("display specific timestamp(default is last modification time) "
+              + "{creation|modification|access}")
+          .build();
+
   private static final Map<String, Comparator<URIStatus>> SORT_FIELD_COMPARATORS = new HashMap<>();
 
   static {
     SORT_FIELD_COMPARATORS.put("creationTime",
         Comparator.comparingLong(URIStatus::getCreationTimeMs));
     SORT_FIELD_COMPARATORS.put("inMemoryPercentage",
         Comparator.comparingLong(URIStatus::getInMemoryPercentage));
+    SORT_FIELD_COMPARATORS.put("lastAccessTime",
```

Comment:
`lastAccessTime` should be added to the description of `SORT_OPTION`

Reply:
Updated.